otelx/middleware_test.go

```diff
@@ -0,0 +1,45 @@
+package otelx
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/urfave/negroni"
+	"go.opentelemetry.io/otel"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/sdk/trace/tracetest"
+)
+
+func TestShouldNotTraceHealthEndpoint(t *testing.T) {
```

Comment:
We should also have a test with a non-health endpoint and check that it gets traced.